src/test/operators/table_scan_test.cpp

```diff
@@ -118,8 +120,9 @@ TEST_F(OperatorsTableScanTest, EmptyResultScan) {
   auto scan_1 = std::make_shared<TableScan>(_table_wrapper, ColumnID{0}, ScanType::OpGreaterThan, 90000);
   scan_1->execute();
 
-  for (auto chunk_index = ChunkID{0}; chunk_index < scan_1->get_output()->chunk_count(); chunk_index++)
+  for (auto chunk_index = ChunkID{0}; chunk_index < scan_1->get_output()->chunk_count(); chunk_index++) {
     EXPECT_EQ(scan_1->get_output()->get_chunk(chunk_index)->column_count(), 2u);
+  }
 }
 
 TEST_F(OperatorsTableScanTest, SingleScanReturnsCorrectRowCount) {
```

Comment:
For test coverage: we are pretty sure more tests are needed to get to 100% or close to it.